tests/python/kivaloo/proto_lbs.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+""" Implentation of the kivaloo communication to LBS. """
+
+import kivaloo.wire
+
+
+class NotBinaryData(TypeError):
+    """ You must pass some `bytes` to this function. """
+    pass
+
+
+class Proto_lbs():
+    """ Wire protocol to a lbs server.  For simplicity, communicates with
+        the server with blocking functions (although the protocol itself is
+        non-blocking).
+    """
+    def __init__(self):
+        super().__init__()
+        self.wire = kivaloo.wire.Wire(kivaloo.servers.LBS_SOCK)
```

Comment:
This means that we can only run connect to a server started by these scripts.  Also, it requires `kivaloo.servers` but doesn't import it.  I'm surprised that pylint didn't complain about this!